doc/articles/wpf-migration.md

```diff
@@ -0,0 +1,60 @@
+# Migrating WPF Apps to Web
+
+This article contains a thorough walkthrough of architecture and implementation considerations for migrating WPF applications to the Web using Uno Platform and WinUI.
+
+![A WPF app migrated to the web with Uno platform](Assets\migrated-app.png)
+
+## Introduction
+
+While improvements will continue to be made by Microsoft at the framework level to ensure the compatibility and reliability of your WPF application, it may be desirable to expand its reach. In this hybrid work era, previous assumptions and development decisions which served your customers well become less valid. For instance, applications targeting the Windows platform alone miss out on mobility benefits of the Web. 
+
+Advancements such as WebAssembly (WASM), a low-level bytecode for the web, have already become a viable, performant way to deliver your modern .NET applications.
+
+## Migrating the codebase
+
+The Uno Platform not only allows you to maximize reuse of C#, XAML, and .NET investments, but also enables your team to naturally leverage their existing skillset. Alongside your existing investments, you may decide to further enhance your app, refreshing aspects to make use of new capabilities in the platform. Regardless, your goal is to consider what role your application will play several years or more into the future. Depending on your scenario, it may make sense to enhance the user interface layout for different form factors. In this article, the primary focus will be on rewriting as little as possible.
+
+> [!TIP]
+> Read the UWP/WinUI & Uno documentation first to better understand the capabilities of the platform. 
+
+## Limitations
+
+The vast majority of your codebase will simply "move across". Existing XAML structure, business logic, and view models can simply be reused. You'll need to recompile old code, and won't have access to framework-only APIs. You may encounter APIs that aren't implemented or controls where a subset of the functionality is presently unsupported. Additionally, the following contain substantial enough differences from WPF to require extra research upfront:
+* Navigation
```

Comment:
Are these planned sections you haven't added yet? Otherwise they should be linked to other docs (either in the Uno docs or Microsoft docs, as appropriate)

Reply:
Although we're now linking to Microsoft documentation for these points, I feel they're all good candidates for us to expand on in the future with Uno-tailored documentation.